LDSAnnotations/SyncAnnotationsOperation.swift

```
@@ -270,7 +270,8 @@ class SyncAnnotationsOperation: Operation {
                                 try annotationStore.addOrUpdateBookmark(downloadedBookmark)
                                 downloadBookmarkCount += 1
                             } else {
-                                throw Error.errorWithCode(.Unknown, failureReason: "Failed to deserialize bookmark: \(bookmark)")
+                                // Don't throw an error, this annotation will eventually come down when an HTML5 version becomes available
```

Can you make this (and the two following cases) throw in all cases except for specifically when the pid is missing but the URI is present? Maybe make the init for these objects throw instead of return nil, and throw 2 different errors for the two cases.